Packs/GenericSQL/Integrations/GenericSQL/GenericSQL.py

```diff
@@ -0,0 +1,199 @@
+    any way defines a different executing way, therefore there are 2 legal return types
+    :param bind_variables_names: the names of the bind variables, must be in the length of the values list
+    :param bind_variables_values: the values of the bind variables, can be in the length of the names list
+            or in case there is no name lists - at any length
+    :return: a dict or lists of the bind variables
+    """
+    bind_variables_names_list = argToList(bind_variables_names)
+    bind_variables_values_list = argToList(bind_variables_values)
+
+    if bind_variables_values and not bind_variables_names:
+        return [var for var in argToList(bind_variables_values)]
+    elif len(bind_variables_names_list) is len(bind_variables_values_list):
+        return dict(zip(bind_variables_names_list, bind_variables_values_list))
+    else:
+        raise Exception("The bind variables lists are not is the same length")
+
+
+def test_module(client: Client, *_) -> Tuple[str, Dict[Any, Any], List[Any]]:
+    """
+    Performs basic connect to SQL server
+    If the connection in the client was successful the test will return OK
+    if it wasn't an exception will be raised
+    """
+    return 'ok', {}, []
+
+
+def sql_query_execute(client: Client, args: dict, *_) -> Tuple[str, Dict[str, Any], List[Dict[str, Any]]]:
+    """
+    Executes the sql query with the connection that was configurator in the client
+    :param client: the client object with the db connection
+    :param args: demisto.args() including the sql query
+    :return: Demisto outputs
+    """
+    try:
+        sql_query = str(args.get('query'))
+        limit = int(args.get('limit', 50))
+        skip = int(args.get('skip', 0))
+        bind_variables_names = args.get('bind_variables_names', "")
+        bind_variables_values = args.get('bind_variables_values', "")
+        bind_variables = generate_bind_vars(bind_variables_names, bind_variables_values)
+
+        result, headers = client.sql_query_execute_request(sql_query, bind_variables)
+        table = [dict(row) for row in result]
+        human_readable = tableToMarkdown(name="Query result:", t=table[skip:skip + limit], headers=headers,
+                                         removeNull=True)
+        context = {
+            'Result': table,
+            'Query': sql_query,
+            'InstanceName': f'{client.dialect}_{client.dbname}'
+        }
+        entry_context: Dict = {f'GenericSQL(val.Query && val.Query === obj.Query)': {'GenericSQL': context}}
+        return human_readable, entry_context, table
+
+    except Exception as err:
+        # explain
+        if str(err) == "This result object does not return rows. It has been closed automatically.":
+            human_readable = "Command executed"
+            return human_readable, {}, []
+        else:
+            return_error(err)
+            return "", {}, []
```

Comment:
no need to return after you `return_error`. you can delete this line as it is unreachable.

Reply:
I know but then I'm getting an error that there is no "return" statement and I can't commit...